scripts/plotParticles.py

```diff
@@ -1,64 +1,86 @@
 #!/usr/bin/env python
 from netCDF4 import Dataset
 import numpy as np
-import matplotlib as mpl
 import matplotlib.pyplot as plt
-from mpl_toolkits.mplot3d import Axes3D
 from argparse import ArgumentParser
 import matplotlib.animation as animation
 
-def particleplotting(filename,tracerfile, mode):
+
+def particleplotting(filename, recordedvar, backgroundfield, mode):
     """Quick and simple plotting of PARCELS trajectories"""
 
-    pfile=Dataset(filename,'r')
+    pfile = Dataset(filename, 'r')
     lon = pfile.variables['lon']
     lat = pfile.variables['lat']
+    time = pfile.variables['time']
     z = pfile.variables['z']
 
-    fig, ax = plt.subplots()
-    if tracerfile != 'none':
-      tfile = Dataset(tracerfile,'r')
-      X = tfile.variables['x']
-      Y = tfile.variables['y']
-      P = tfile.variables['P']
-      plt.contourf(np.squeeze(X),np.squeeze(Y),np.squeeze(P))
+    if(recordedvar is not 'none'):
+        rMax = np.max(pfile.variables[recordedvar])
+        record = pfile.variables[recordedvar]
+
+    if backgroundfield is not 'none':
+        bfile = Dataset(backgroundfield, 'r')
+        bX = bfile.variables['x']
+        bY = bfile.variables['y']
+        bT = bfile.variables['time_counter']
```

Comment:
Why not use the same functions here as we use in the main parcels code? Can we use the `Grid.from_netcdf `function? That would also allow users to use the `variables = {}` and `dimensions={}` dictionaries in a similar way as in the main PARCELS code and make updating codebases much easier.